packages/components/atoms/f-popover/src/components/Popover.vue

```diff
@@ -26,18 +26,6 @@ $tooltip-width                 : 10px;
         border-radius: $radius-rounded-c;
         padding: 0 $popover-padding;
         width: auto;
```

Comment:
Line 20 contains `$tooltip-width` that seems to be made redundant with the change